bangazon-clumsy-penguins/WorkforceManagement
# Sd employees

# Employees - Details

## Description
Provides details view for Employees.

##### Provide clarity to any potentially complex code:


## Steps to Test
Run in VS, click employees in navigation bar, click details for any employee. Verify all information is correct including firstname, lastname, department, computer, and trainings.

##### Link to Feature Ticket
#5
